hammett/core/screen.py

```diff
@@ -366,6 +366,42 @@ async def get_callback_query(update: 'Update') -> 'CallbackQuery | None':
 
         return query
 
+    def get_description(
```

Comment:
Make all methods async because we'll can make request to some backend to get description